pkg/models/base_task_list_section_user_table.go

```diff
@@ -0,0 +1,117 @@
+package models
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/mint-app/pkg/appcontext"
+	"github.com/cmsgov/mint-app/pkg/authentication"
+)
+
+// baseTaskListSection represents all the shared fields in common to a task list section
+type baseTaskListSectionUserTable struct {
+	baseStructUserTable
+	modelPlanRelation
+	ReadyForReviewBy     *uuid.UUID `json:"readyForReviewBy" db:"ready_for_review_by"`
+	ReadyForReviewDts    *time.Time `json:"readyForReviewDts" db:"ready_for_review_dts"`
+	ReadyForClearanceBy  *uuid.UUID `json:"readyForClearanceBy" db:"ready_for_clearance_by"`
+	ReadyForClearanceDts *time.Time `json:"readyForClearanceDts" db:"ready_for_clearance_dts"`
+	Status               TaskStatus `json:"status" db:"status"`
+}
+
+// NewBaseTaskListSectionUserTable makes a task list section by a modelPlanID and user id of the user creating it
+func NewBaseTaskListSectionUserTable(createdBy uuid.UUID, modelPlanID uuid.UUID) baseTaskListSectionUserTable {
+
+	return baseTaskListSectionUserTable{
+		modelPlanRelation: NewModelPlanRelation(modelPlanID),
+
+		Status:              TaskReady,
+		baseStructUserTable: NewBaseStructUser(createdBy),
+	}
+
+}
+
+// GetBaseTaskListSection returns the BaseTaskListSection Object embedded in the struct
+func (b *baseTaskListSectionUserTable) GetBaseTaskListSectionUserTable() *baseTaskListSectionUserTable {
+	return b
+}
+
+// GetStatus returns the status of a basesTaskListSection
+func (b *baseTaskListSectionUserTable) GetStatus() TaskStatus {
+	return b.Status
+}
+
+// // GetBaseStruct returns the Base Struct
+// func (b *baseTaskListSectionUserTable) GetBaseStruct() *baseStruct {
+// 	return b.baseStructUserTable
+// }
```

Comment:
Do we need to keep this comment?

Reply:
Good call! I think it should be removed.